stdlib/SparseArrays/test/sparse.jl

```diff
@@ -2242,4 +2242,10 @@ _length_or_count_or_five(x) = length(x)
     end
 end
 
+@testset "sparse transpose adjoint" begin
+    A = sprand(10, 10, 0.75)
+    @test A' == SparseMatrixCSC(A')
```

Comment:
We might want to verify the return type, too, because the new definitions do not _obviously_ return a `SparseMatrixCSC`.

Reply:
Done